### ISSUE TITLE
Examples for multiple treatments 

### ISSUE THREAD

**Issue by yolimonsta** [Ibotta/mr_uplift#46]
Hi! 

It is really exciting to see there is a package for multiple treatment uplift models. Wonder would it be possible to add an example for the multiple treatments model. 

I found this 'This means that a user can input an arbitrary number of treatments (not just the usual binary variable) into the model' in your blog plots, but I am not so sure what kind of input is expected. For the multiple treatments, does it only allow the number of treatments or does it also allow a series of binary features? 

Best,
Yolanda

**Comment by samcarlos** [comment]
Hey Yolanda, 

Thanks for your question. Basically you can include any numbers of treatments and encode them any way you'd like as long as they're numeric. 

For instance in [this blog post](http://scweiss.blogspot.com/2020/06/the-mruplift-package-in-r-practitioners.html) I have a list of treatments encoded:

`[[0.0,0.0,0.0,0.0],
   [1.0,0.0,0.0,0.0],
   [1.0,1.0,0.0,0.0],
   [1.0,1.0,1.0,0.0]]`

Each row is a treatment and each column is a feature that some treatments have in common. Also note that these can be real numbers not just ones and zeros. 

If you have list of unique treatments the treatments list could be a dummied-out matrix. 

Does that answer your question? 

Thanks,
Sam

**Comment by yolimonsta** [comment]
Thanks for sharing that! It is very helpful. For the multiple treatment models, are you implementing the algorithms from Uplift Modeling with Multiple Treatments and General Response Types? 

Thanks!
Yolanda

**Comment by samcarlos** [comment]
Hey Yolanda, 

I use the same out-of-sample metric they use in that paper in [section 2.2](https://arxiv.org/pdf/1705.08492.pdf). However, I use a neural network and custom loss function to estimate the model. You can find more information about that [here](https://github.com/Ibotta/mr_uplift/blob/master/examples/mr_uplift_new_optimized_loss.ipynb). 

Thanks,
Sam

**Comment by yolimonsta** [comment]
Thanks for the reply! It is very helpful.

Thanks,
Yolanda

**Comment by samcarlos** [comment]
Hey Yolanda, 

Can I close the issue? 

Thanks,
Sam

**Comment by yolimonsta** [comment]
Yes please!

Thanks a lot.
Yolanda
